ws-nextjs-app/pages/[service]/new_live/[id]/[[...variant]].page.tsx

```diff
@@ -96,11 +97,26 @@ export const getServerSideProps: GetServerSideProps = async context => {
     service,
     variant,
     // renderer_env: rendererEnv,
-    page,
+    page = '1',
   } = context.query as PageDataParams;
 
   const { headers: reqHeaders } = context.req;
 
+  if (!isValidPageNumber(page)) {
```

Comment:
Will this error if no page number is given? I feel like the URL on the initial request won't have the `page` query param

Reply:
Actually ignore! I missed the `page` getting set to 1 by default in the `context.params` destruct 🙃